src/webprojects/fxa.md

```diff
@@ -5,13 +5,14 @@ Firefox Accounts (or FxA) refers to [https://accounts.firefox.com](https://accou
 * `LC_MESSAGES/client.po`: the content server for UI and frontend.
 * `LC_MESSAGES/server.po`: strings for emails and backend templates.
 * `Main.ftl`: payments for both frontend and backend.
+* `Settings.ftl`: new subscription service.
```

Comment:
I understood this is used for the account's redesigned settings in general, not just subscription?